src/test/java/fi/helsinki/cs/tmc/cli/io/ExternalUtilTest.java

```diff
@@ -23,7 +9,7 @@
 import org.powermock.core.classloader.annotations.PrepareForTest;
 import org.powermock.modules.junit4.PowerMockRunner;
 
-import java.awt.Desktop;
+import java.awt.*;
```

Comment:
Avoid star imports

Reply:
It seems like IntelliJ offers star imports, I will stop using the Alt+Enter auto import functionality.